admin_guide/iptables.adoc

```diff
@@ -11,18 +11,76 @@
 toc::[]
 
 == Overview
-This topic describes how administrators should work with iptables. openshift-sdn takes care of adding the
-necessary iptables rules to make it work. Kubernetes and Docker also manage iptables for port forwarding
-and services.
+There are many system components including {product-title}, Docker, and software that manages local
+firewall policies that rely on the kernel iptables configuration for proper network operation. In
+addition the iptables configuration of all nodes in the cluster must be correct for networking to work.
 
+All components independently work with iptables without knowledge of how other components
+are using them. This makes it very easy for one component to break another component's configuration.
+Further, {product-title} and Docker assume that iptables remains set up exactly as they have set it up.
+They may not detect changes introduced by other components and if they do there maybe some lag in
+implementing the fix. In particular, {product-title} does monitor and fix problems, however, Docker does not.
 
-== Restarting
-Docker doesn't monitor the iptables rules that it adds for exposing ports from containers and hence if
-iptables service is restarted, then these rules are lost. So, to safely restart iptables, it is
-recommended that the rules are saved and restored. 
+IMPORTANT: You must make sure that whatever changes you make to the iptables configuration on a node
+do not impact the operation of {product-title} and Docker. Also, changes will often need to be made on
+all nodes in the cluster. Be careful, iptables is not designed to have multiple concurrent users and it
+is very easy to break {product-title} and Docker networking.
+
+The chains, order of the chains and rules in the kernel iptables must be properly set up on each node
+in the cluster for {product-title} and Docker networking to work properly. There are several tools and
+services that are commonly used in the system that interact with the kernel iptables and can
+accidentally impact {product-title} and Docker.
+
+=== iptables
+`iptables` is a low level tool that can be used to set up, maintain, and inspect the tables of IPv4
+packet filter rules in the Linux kernel.
+
+WARNING: This is a very powerful tool that can manage all aspects of the kernel iptables. It is very
+easy to break networking in difficult to debug ways so be careful.
```

Comment:
I still think this section is misleading; as far as I remember, *all* of the iptables-related problems we have seen have been caused by careless use of iptables.service, not by using /sbin/iptables directly.